### DISCUSSION TITLE
Notepad++ Support

### DISCUSSION THREAD

**Discussion by PierrunoYT** [community/community#48990] [Copilot Conversations] [answered]
### Select Topic Area

Product Feedback

### Body

Why doesn't Copilot support Notepad++? It is much easier to use than Visual Studio, even for people without much programming knowledge.

**Comment by SilasRodrigues19** [comment, marked as answer]
Hey @PierrunoYT 

There is an answer about this in this other discussion, at the following link:

https://github.com/orgs/community/discussions/19408#discussioncomment-3024086
